var/spack/repos/builtin/packages/postgresql/package.py

```diff
@@ -125,3 +125,22 @@ def setup_dependent_run_environment(self, env, dependent_spec):
             env.prepend_path('TCLLIBPATH', self.prefix.lib)
         if '+python' in spec:
             env.prepend_path('PYTHONPATH', self.prefix.lib)
+
+    @property
+    def libs(self):
+        stat_libs = ['libecpg_compat', 'libecpg', 'libpgcommon',
+                     'libpgcommon_shlib', 'libpgfeutils', 'libpgport',
+                     'libpgport_shlib', 'libpgtypes', 'libpq']
+        fl_stat = find_libraries(stat_libs, self.prefix, shared=False,
+                                 recursive=True)
+
+        dyn_libs = ['libecpg_compat', 'libecpg', 'libpgtypes', 'libpq',
+                    'libpqwalreceiver', 'plpgsql', 'pgoutput']
+        dyn_libs.extend('plperl') if '+perl' in self.spec
+        dyn_libs.extend('plpython') if '+python' in self.spec
+        dyn_libs.extend('pltcl') if '+tcl' in self.spec
```

Comment:
This still seems to be using extend instead of append. Also, is this even valid Python?

Reply:
Ooops. I thought I already fixed it. It seemed to work just fine in python2. But append is the way to go.